src/mail-app/mail/view/ConversationViewModel.ts

```diff
@@ -302,7 +302,14 @@ export class ConversationViewModel {
 		return isOfTypeOrSubfolderOf(folders, mailFolder, MailSetKind.TRASH)
 	}
 
-	conversationItems(): ReadonlyArray<ConversationItem> {
+	readonly conversationMails: () => readonly Mail[] = memoizedWithHiddenArgument(
```

Comment:
I'm not sure if it's a good idea to memoize like this, mail instance on `viewModel` might get updated, in this ViewModel we only react to `ConversationEntry` updates and the rest is inside the `MailViewerViewModel`. If we memoize base on items we will memoize outdated mails